deepreg/dataset/loader/interface.py

```diff
@@ -120,16 +125,32 @@ def get_dataset_and_preprocess(
         dataset = dataset.batch(batch_size=batch_size, drop_remainder=training)
         dataset = dataset.prefetch(tf.data.experimental.AUTOTUNE)
         if training:
-            # TODO add cropping, but crop first or rotation first?
-            affine_transform = AffineTransformation3D(
+            da_transform = AffineTransformation3D(
```

Comment:
ideally, it'd better to have a `get_transform` function, which returns one single function (inside which you can call multiple functions), then we apply it for training only,
then depending on the configs, we can choose to apply affine transform, or ddf transform

So I suggest to
1. define a `build_transform` or `get_transform` function
2. change the config for ddf only for now, as your da config is not covering affine

what do you think @acasamitjana 

Ideally, if reviews go faster, I should change all register system before mid Dec, then the whole DA system will be easier